src/quickgui/qgsquickutils.h

```diff
@@ -124,10 +124,13 @@ class QUICK_EXPORT QgsQuickUtils: public QObject
     Q_INVOKABLE static bool fileExists( const QString &path );
 
     /**
-     * Extracts filename from path
+     * Returns filename from given path. If a param prefixPath is not empty, returns relative path of the file to this path.
+     * If a path starts with "file://", this prefix is ignored.
+     * @param path Absolute path to file
+     * @param prefixPath Optional parameter; defines if the function returns fileName or a relative path to this path.
      * \since QGIS 3.4
      */
-    Q_INVOKABLE static QString getFileName( const QString &path );
+    Q_INVOKABLE static QString getFileName( const QString &path, const QString &prefixPath = QStringLiteral( "" ) );
```

Comment:
can we keep getFileName() as is and have a new function relativePath(path, prefixPath)

Reply:
(and maybe get rid of getFileName() if it is not needed anymore?)